logfire/_internal/metrics.py

```diff
@@ -44,20 +44,15 @@ def get_meter(
         name: str,
         version: str | None = None,
         schema_url: str | None = None,
-        *args: Any,
-        **kwargs: Any,
+        attributes: Attributes | None = None,
     ) -> Meter:
         with self.lock:
             if name in self.suppressed_scopes:
                 provider = NoOpMeterProvider()
             else:
                 provider = self.provider
-            meter = _ProxyMeter(
-                provider.get_meter(name, version=version, schema_url=schema_url, *args, **kwargs),
-                name,
-                version,
-                schema_url,
-            )
+            inner_meter = provider.get_meter(name, version, schema_url, attributes)
```

Comment:
This breaks OTEL<=1.25.0. For now I'm OK with that. The tests fail in other ways in 1.26.0 (though things might still be working).